faqs.md

```diff
@@ -1 +1,47 @@
-## FAQs
\ No newline at end of file
+## FAQs
+
+### What is the OpenMRS Android app?
+
+You can find out what the OpenMRS Android app is and the motivation behind the creation of this app [here](https://github.com/openmrs/openmrs-android-client-user-guide/blob/master/introduction.md).
+
+### How do I get started?
+
+You can find information about installing and using the app [here](https://github.com/openmrs/openmrs-android-client-user-guide/blob/master/getting-started.md).
```

Comment:
Change to https://openmrs.github.io/openmrs-android-client-user-guide/getting-started.html